src/motor_python.cpp

```diff
@@ -188,38 +189,38 @@ PYBIND11_MODULE(motor, m)
         .def("set_command_position_tuning", &MotorManager::set_command_position_tuning)
         .def("set_command_current_tuning", &MotorManager::set_command_current_tuning);
 
-    py::class_<Motor, std::shared_ptr<Motor>>(m, "Motor")
+    py::class_<MotorUSB, std::shared_ptr<MotorUSB>>(m, "MotorUSB")
```

Comment:
I will look at this in more detail. One quick comment is can we keep this interface for Motor? Most are all things I'd ideally like to always be able to pass through. Devnum and dev_path are a couple that are more specific. I repurposed them in MotorIP though.

Reply:
I had to change this since `Motor` is now an abstract base class and can't be instantiated by Pybind. I'm not clear on how you're using this Python binding though. If you point me to some example usage, maybe I can find a good compromise for this.